apps/cli/src/commands/profile/update.profile.ts

```diff
@@ -57,7 +54,12 @@ export default class UpdateProfile extends BaseCommand {
     s.start('Updating the profile')
 
     checkProfileExists(this.profiles, profile, s)
-    this.updateProfileData(profile, name, apiKey, baseUrl)
+    this.updateProfileData(
```

Comment:
I feel instead of type casting the values in here to `string`, we can update the function signature of the `updateProfileData` method to have `?` after each of the fields.

Reply:
I think typecasting will give a better clarity to the function

Reply:
True that, but as per the use case, we can't expect all the values to be defined. And thats why, we are doing a conditional update. Thats the reason i asked for the change

Reply:
this values are marked as any